src/pages/callback.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { handleAuthentication } from "../utils/auth"
+import { navigate } from "gatsby"
+
+const Callback = () => {
+  handleAuthentication(() => navigate("/account"))
+
+  return <p>Callback</p>
+}
+
```

Comment:
### 8. {Handle the oauth callback route}

> Lines {5} - {9}

Here we simply call our previously-defined `handleAuthentication` method, which parses and verifies the token before redirecting- in our case, to `/account`.

[Next →](https://github.com/erikdstock/gatsby-starter-ts-auth0/pull/2#pullrequestreview-239278342)